skyux-spa-ui/src/app/app-sky.module.ts

```diff
@@ -21,9 +21,10 @@ import {
   SkyCheckboxModule
 } from '@skyux/forms';
 import { SkyI18nModule } from '@skyux/i18n';
+import { SkyIconModule } from '@skyux/icon';
```

Comment:
`SkyIconModule` is being removed.  Is `import { SkyIconModule } from '@skyux/icon';` needed?

Reply:
Removed